README.md

```diff
@@ -46,3 +46,14 @@ The backend consists of three microservices
 
 With this setup the lambda functions are configured to automatically hot reload. This means that if you perform a change
 in e.g findme-users/ you will instantly see the change after saving.
+
+
+### Token
+To get a token for developing use the following curl command:
+`curl --request POST \
+  --url https://findme-dev.eu.auth0.com/oauth/token \
+  --header 'content-type: application/json' \
+  --data '{"client_id":"0FhpaZeIjhSG1lwNR3RWPI20VgLgU5rk",
+            "client_secret":"XU4YFkurDj5v2L6X7crdB5grRxC_H3HzSk18rKbYJaq3b9M5Vbc6GnfP2c4PeD8v",
```

Comment:
Not sure we should expose our client secret like this, oh well I guess its too late now 😉